torchvision/transforms/v2/functional/__init__.py

```diff
@@ -1,6 +1,6 @@
 from torchvision.transforms import InterpolationMode  # usort: skip
 
-from ._utils import is_simple_tensor  # usort: skip
+from ._utils import is_simple_tensor, register_kernel  # usort: skip
```

Comment:
If this is the helper we expose to users, why not put it in the `v2` namespace directly? It's not really related to our functionals as far as users are concerned (user = someone registering a kernel). No strong opinion

Reply:
Are you ok with exposing it in both namespaces similar to what we do with `InterpolationMode`.

https://github.com/pytorch/vision/blob/8233c9cdf3351e1996249fdb4f3a998f8c9e693d/torchvision/transforms/v2/__init__.py#L1

https://github.com/pytorch/vision/blob/8233c9cdf3351e1996249fdb4f3a998f8c9e693d/torchvision/transforms/v2/functional/__init__.py#L1

Reply:
For InterpolationMode it makes sense because to expose it twice because it's needed for both transforms and functionals. But this is a bit different here. I'd rather keep in `functional` than to expose is in 2 locations TBH.

Reply:
If we don't want it in two places, I prefer to leave it where it is. The `@register_kernel` decorator needs a dispatcher as input. Thus, even if we expose the decorator in the `transforms` namespace, users still need to have `functional` imported to use it. OTOH, if we leave it in `functional`, the user doesn't need to deal with `transforms` when only wanting to register a new datapoint.